module/VuFindApi/config/module.config.php

```diff
@@ -12,6 +12,12 @@
             'SearchApi' => 'VuFindApi\Controller\SearchApiController',
         ],
     ],
+    'service_manager' => [
+        'allow_override' => true,
```

Comment:
Do you need this allow_override setting here? Won't it be inherited from the main VuFind module?

Reply:
I wasn't sure about that, but in any case if VuFindApi somehow became less dependent on the main module in the future, I think it wouldn't hurt to be explicit here. Do you think it would be better without?